test/docker.js

```diff
@@ -0,0 +1,14 @@
+const exec = require('child_process').exec;
```

Comment:
It would become complex as we would have to handle `apt-get`, `brew`, and other package managers. If prefer to have it in the project requirements, as for nodejs and make